src/pages/stacks-blockchain/mining.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Understand Mining
+description: A guide to mining on Stacks 2.0
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Introduction
+
+This guide highlights some technical details related to minig on the Stacks 2.0 network.
+
+## Mining frequency
+
+A new Stacks block may be mined once per Bitcoin block. To be considered for mining a block, a miner must have a block commit included in a Bitcoin block. If a miner wishes to update their commitment after submission, they may use Bitcoin Replace-By-Fee.
+
+## Coinbase Rewards
+
+Miners receive coinbase rewards for blocks they mine. These rewards are shared (proportional to the amount committed to) with an user burn support operations that support the block.
+
+The reward amounts are:
+
+- 1000 STX per block are released in the first 4 years of mining
+- 500 STX per block are released during the following 4 years
+- 250 Stacks tokens per block are released during the following 4 years
+- 125 STX per block are released from then on indefinitely.
+
+These "halvings" are synchronized with Bitcoin halvings.
+
+![coinbase rewards](/images/pages/coinbase-rewards.png)
+
+## Transaction fees
+
+Miners receives Stacks fees for transactions mined in any block they produce.
+
+For transactions mined in microblocks, the miner that produces the microblock receives 40% of the fees, while the miner that confirms the microblock receives 60% of the fees.
+
+## Reward maturity
+
+Miner rewards take 100 blocks to mature.
```

Comment:
Miner rewards, i.e. block rewards and transaction fees take 100 blocks on the bitcon blockchain to mature. After successful mining a block your rewards will appear in the stacks account after ~24 hours.